src/FocusedProofreading.jsx

```diff
@@ -474,6 +474,12 @@ function FocusedProofreading(props) {
   const handleHelpClose = () => { setHelpOpen(false); };
 
   const handleKeyPress = (event) => {
+    // Ignore keyboard shortcuts when a Neuroglancer text input has focus.
+    const focus = document.activeElement;
+    if (focus && focus.type && focus.type.includes('text')) {
```

Comment:
I don't think this covers all the HTML5 input elements, that we might encounter, such as `<input type="email">`.
How about this?

```javascript
if (focus && (/input|textarea/i.test(focus.tagName) || focus.getAttribute('contenteditable') !== null)) {
   return
}
```

Reply:
Your new test covers too much.  For example, a radio button has `tagName` of `'INPUT'` but we don't want to disable keyboard shortcuts when a radio button was the last control used.  My original test distinguished that case because a radio button's `type` is `'radio'` and does not include `'text'`.

Maybe we could just add the `focus.getAttribute('contenteditable') !== null)` part to what I had before?  I'll experiment.

Reply:
I guess we need to define which input elements are safe and which ones aren't it might be easier to check for all inputs and ignore radio or select boxes but disable key press for all others. Alternatively if we are sure that input types of number, email, slider, date, and url wont be used, then we can stick with your original version.